x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/actions_column/actions_column.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { euiThemeVars } from '@kbn/ui-theme';
+
+const DEFAULT_ACTION_BUTTON_WIDTH = 32;
+export const MIN_ACTION_COLUMN_HEADER_WIDTH = 75;
+
+/**
+ * Returns the width of the Actions column based on the number of buttons being
+ * displayed
+ *
+ * NOTE: This function is necessary because `width` is a required property of
+ * the `EuiDataGridControlColumn` interface, so it must be calculated before
+ * content is rendered. (The width of a `EuiDataGridControlColumn` does not
+ * automatically size itself to fit all the content.)
+ */
+export const getActionsColumnWidth = (actionButtonCount: number): number => {
```

Comment:
Hey @elastic/response-ops-ram !

I have some questions regarding the function in the current line and https://github.com/elastic/kibana/blob/main/x-pack/plugins/timelines/public/components/t_grid/body/column_headers/helpers.tsx#L123-L144 as I would like to merge both of these helper functions.

1. What is gonna happen to the t_grid component? It sounds like it should tend to disappear slowly as we want to deprecate it in favour of EuiDataGrid. If so, I'd rather prefer to have this `getActionColumnWidth` and duplicate code that is used in the timelines' helper `getActionColumnWidth` function 
2. In case it's not being deprecated. We could update the timelines' helper fn easily to add the functionality we need here but when trying to use the timeline plugin adding it to the `kibana.json` as `requiredPlugin` I get this error:

 >FATAL  Error: Topological ordering of plugins did not complete, these plugins have cyclic or missing dependencies: ["discover","apm","cases","crossClusterReplication","dataVisualizer","discoverEnhanced","enterpriseSearch","fleet","indexLifecycleManagement","indexManagement","infra","kubernetesSecurity","lens","logstash","maps","ml","monitoring","observability","osquery","remoteClusters","rollup","ruleRegistry","securitySolution","sessionView","stackAlerts","synthetics","timelines","transform","triggersActionsUi","upgradeAssistant","ux"]

And I'd need some help here to solve this 🆘

Reply:
I think that we can duplicate this function for sure because we want slowly to deprecate the alert table form the timeline plugins.

Reply:
Timelines is used outside of alerts as well, so perhaps it makes sense to move shared helpers to a package or something

Reply:
we will let you manage it ;)